### ISSUE TITLE
Cannot stop an experiment with the status of PLAYING

### ISSUE THREAD

**Issue by disperse** [gallup/NGS2-Router#17]
Steps to reproduce:

1. Start a new experiment instance
2. Join enough clients so the game begins
3. Click the Edit button next to the Experiment Instance
4. Click the Stop button
5. Click the Check button

Expected results: The experiment instance's status will change to STOPPED

```
Actual results: The following server-side error is thrown:

! @7527b5ln6 - Internal server error, for (POST) [/app/updateExperimentInstance/3] ->
 
play.api.http.HttpErrorHandlerExceptions$$anon$1: Execution exception[[DataIntegrityException: Error when batch flush on sql: insert into user_info_experiment_instance (experiment_instance_id, user_info_id) values (?, ?)]]
        at play.api.http.HttpErrorHandlerExceptions$.throwableToUsefulException(HttpErrorHandler.scala:255)
        at play.api.http.DefaultHttpErrorHandler.onServerError(HttpErrorHandler.scala:180)
        at play.core.server.AkkaHttpServer$$anonfun$13$$anonfun$apply$1.applyOrElse(AkkaHttpServer.scala:252)
        at play.core.server.AkkaHttpServer$$anonfun$13$$anonfun$apply$1.applyOrElse(AkkaHttpServer.scala:251)
        at scala.concurrent.Future$$anonfun$recoverWith$1.apply(Future.scala:346)
        at scala.concurrent.Future$$anonfun$recoverWith$1.apply(Future.scala:345)
        at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:36)
        at play.api.libs.streams.Execution$trampoline$.executeScheduled(Execution.scala:109)
        at play.api.libs.streams.Execution$trampoline$.execute(Execution.scala:71)
        at scala.concurrent.impl.CallbackRunnable.executeWithValue(Promise.scala:44)
Caused by: io.ebean.DataIntegrityException: Error when batch flush on sql: insert into user_info_experiment_instance (experiment_instance_id, user_info_id) values (?, ?)
        at io.ebean.config.dbplatform.SqlCodeTranslator.translate(SqlCodeTranslator.java:44)
        at io.ebean.config.dbplatform.DatabasePlatform.translate(DatabasePlatform.java:188)
        at io.ebeaninternal.server.transaction.TransactionManager.translate(TransactionManager.java:145)
        at io.ebeaninternal.server.transaction.JdbcTransaction.translate(JdbcTransaction.java:651)
        at io.ebeaninternal.server.transaction.JdbcTransaction.batchFlush(JdbcTransaction.java:634)
        at io.ebeaninternal.server.transaction.JdbcTransaction.internalBatchFlush(JdbcTransaction.java:749)
        at io.ebeaninternal.server.transaction.JdbcTransaction.flushCommitAndNotify(JdbcTransaction.java:934)
        at io.ebeaninternal.server.transaction.JdbcTransaction.commit(JdbcTransaction.java:986)
        at io.ebeaninternal.server.core.BeanRequest.commitTransIfRequired(BeanRequest.java:61)
        at io.ebeaninternal.server.persist.DefaultPersister.update(DefaultPersister.java:375)
Caused by: org.h2.jdbc.JdbcBatchUpdateException: NULL not allowed for column "ARRIVAL_TIME"; SQL statement:
insert into user_info_experiment_instance (experiment_instance_id, user_info_id) values (?, ?) [23502-193]
        at org.h2.jdbc.JdbcPreparedStatement.executeBatch(JdbcPreparedStatement.java:1208)
        at com.zaxxer.hikari.pool.ProxyStatement.executeBatch(ProxyStatement.java:125)
        at com.zaxxer.hikari.pool.HikariProxyPreparedStatement.executeBatch(HikariProxyPreparedStatement.java)
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:497)
        at org.jdbcdslog.PreparedStatementLoggingHandler.invoke(PreparedStatementLoggingHandler.java:35)
        at com.sun.proxy.$Proxy78.executeBatch(Unknown Source)
        at io.ebeaninternal.server.persist.BatchedPstmt.executeAndCheckRowCounts(BatchedPstmt.java:103)

```

**Comment by uduwage** [comment]
@disperse  Could this be annotation related tried various implementations but @ManyToMany complains while updates of dependent tables.

**Comment by disperse** [comment]
Fixed with commit bf1c9e3f8863e18c30dd49db3e2d67adfdce91da
